client/settings-view/templates/inferenceSettingsView.pug

```diff
@@ -0,0 +1,207 @@
+
+          a.nav-link.tab.active(data-hook=this.model.elementID + "-inference-settings-edit-tab" data-toggle="tab" href="#" + this.model.elementID + "-edit-inference-settings") Edit
+
+        li.nav-item
+
+          a.nav-link.tab(data-hook=this.model.elementID + "-inference-settings-view-tab" data-toggle="tab" href="#" + this.model.elementID + "-view-inference-settings") View
+
+    button.btn.btn-outline-collapse(data-toggle="collapse" data-target="#" + this.model.elementID + "collapse-inference-settings" data-hook="collapse") -
+
+  div.collapse(class="show" id=this.model.elementID + "collapse-inference-settings")
+
+    div.card-body.tab-content
+
+      div.tab-pane.active(id=this.model.elementID + "-edit-inference-settings" data-hook=this.model.elementID + "-edit-inference-settings")
+
+        div
+
+          hr
+
+          div.mx-1.row.head.align-items-baseline
+
+            div.col-sm-3
+
+              h6.inline <b>Number of Rounds</b>
+
+            div.col-sm-3
+
+              h6.inline <b>Number of Samples</b>
+
+          div.mx-1.my-3.row
+
+            div.col-sm-3(data-hook="num-rounds")
+
+            div.col-sm-3(data-hook="num-samples")
+
+        div
+
+          h4 Summary Statistics
+
+          hr
+
+          div.mr-3.inline
+
+            span.mr-2.inline(for="#summaryStatsTypeSelect")
+
+              h6 <b>Summary Statistics Type</b>:
+
+            div.inline(id="summaryStatsTypeSelect" data-hook="summary-stats-type-select")
+
+          div.inline.hidden(data-hook="tsfresh-docs-link")
+
+            p See <a href="https://tsfresh.readthedocs.io/en/latest/api/tsfresh.feature_extraction.html#module-tsfresh.feature_extraction.feature_calculators" target="_blank">TSFresh documentation</a> for a full list of supported feature calculators.
+
+          div(data-hook="edit-summary-stats-container")
+
+        div.my-3(data-hook="edit-parameter-space-container")
+
+        div.accordion(id="obs-data-section" data-hook="obs-data-section")
+
+          div.card
+
+            div.card-header(id="importObsDataHeader")
+
+              h4.mb-0.inline="Import Observed Data File"
+
+              button.btn.btn-outline-collapse.collapsed.inline(type="button" data-toggle="collapse" data-target="#collapseImportObsData" data-hook="collapseImportObsData" aria-expanded="false" aria-controls="collapseImportObsData" style="float: right") 
+
+                div(data-hook="import-chevron")
+
+                  svg(xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" viewBox="0 0 512 512")
+
+                    path(d="M233.4 406.6c12.5 12.5 32.8 12.5 45.3 0l192-192c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L256 338.7 86.6 169.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3l192 192z")
+
+            div.card-body
+
+              p.mb-0
+                | Valid files are <b>Observed Data Files</b> (a directory with the postfix .odf) compressed to a zip archive (.zip) or <b>Observed Data File</b> (.csv)
```

Comment:
I do think this is a little weird... the Bold followed by the parenthesis implies a relationship between the bolded words and the extension.... but .csv is not technically connected to "Observed Data File"